app/models/identity.rb

```diff
@@ -15,15 +15,9 @@ class Identity < ActiveRecord::Base
 
   delegate :tasks_count, :unaccessed_documents_count, to: :identity_counter
 
+
   def protocols
-    if clinical_providers.any?
-      clinical_providers.
-        map { |clinical_provider| clinical_provider.organization.protocols }.
-        compact.
-        flatten
-    else
-      Array.new
-    end
+    fulfillment_organizations_with_protocols.any? ? fulfillment_organizations_with_protocols.map(&:protocols).flatten : []
```

Comment:
fulfillment_organizations_with_protocols.map(&:protocols) should either return protocols or empty [].  I'm not sure the check is necessary.